test/test_shapes.py

```diff
@@ -104,13 +104,8 @@ def test_n(self):
         assert t.shape[0] == 345
 
     def test_rotation(self):
-        try:
+        with pytest.raises(AssertionError) as ae:
```

Comment:
This will only catch the first one.  I don't think the second call with angle=300 will ever hit the `pytest.raises` if the first one throws.  Could you wrap both in their own `pytest.raises` block?

Reply:
Yes! Absolutely!

Reply:
Done!